sgv2-restapi/pom.xml

```diff
@@ -262,7 +262,7 @@
             </goals>
             <configuration>
               <transformers>
-                <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer" />
```

Comment:
nit: looks like an accidental change

Reply:
That's odd. I'll revert that manually.

Reply:
Maybe some kind of auto-formatter? Both are legal, no semantic difference, but maybe enforced style?

Reply:
actually will leave this for now, let's see what happens on the next build.